common/bootstrap/templates/coredns.yaml

```
@@ -28,6 +28,17 @@ data:
         rewrite name 9c-main-validator-8.nine-chronicles.com validator-8.9c-network.svc.cluster.local
         rewrite name 9c-main-test-1.nine-chronicles.com test-headless-1.9c-network.svc.cluster.local
         rewrite name 9c-main-test-2.nine-chronicles.com test-headless-2.9c-network.svc.cluster.local
+        rewrite name heimdall-seed-1.nine-chronicles.com tcp-seed-1.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-1.nine-chronicles.com remote-headless-1.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-2.nine-chronicles.com remote-headless-2.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-3.nine-chronicles.com remote-headless-3.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-4.nine-chronicles.com remote-headless-4.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-5.nine-chronicles.com remote-headless-5.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-6.nine-chronicles.com remote-headless-6.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-7.nine-chronicles.com remote-headless-7.heimdall.svc.cluster.local
+        rewrite name heimdall-rpc-8.nine-chronicles.com remote-headless-8.heimdall.svc.cluster.local
+        rewrite name heimdall-validator-5.nine-chronicles.com validator-5.heimdall.svc.cluster.local
```

```suggestion
        rewrite name heimdall-validator-1.nine-chronicles.com validator-5.heimdall.svc.cluster.local
```

Could you add full state too?